File: 1주차/21년01월03일/이정현/9498.js
const input = require("fs").readFileSync("/dev/stdin").toString().split(" ");

const a = Number(input);

if(a >= 90) {
    console.log("A")
} else if(89 >= a && a >= 80) {
    console.log("B")
} else if(79 >= a && a >= 70) {
    console.log("C")
} else if(69 >= a && a >= 60 ) {
    console.log("D")
} else {
    console.log("F")
};

File: 1주차/21년01월03일/박광현/9498.swift

let input = readLine()!
let inputArr = input.split(separator:" ")
let a = Int(inputArr[0])!

if a >= 90 {
    print("A")
}else if a >= 80 {
    print("B")
}else if a >= 70 {
    print("C")
}else if a >= 60 {
    print("D")
}else if a < 60 {
   print("F")
}
